indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerMapper.java

```diff
@@ -104,6 +105,10 @@ public final static InputRow parseInputRow(Object value, InputRowParser parser)
     if(parser instanceof StringInputRowParser && value instanceof Text) {
       //Note: This is to ensure backward compatibility with 0.7.0 and before
       return ((StringInputRowParser) parser).parse(value.toString());
+    } else if(parser instanceof StringInputRowParser && value instanceof BytesWritable) {
```

Comment:
@gianm you are right about the deserialization issue. (but users can still do that as long as they put hadoop libs on the classpath and it will work but we can't put it in core druid as it does not work in all cases)
If adding this support is a very general usecase needed by many users then it might be ok to let special handling happen for now while we can work out a model where it is feasible to write parsers that depend on hadoop libs and work in all cases.